eth-types/src/bytecode.rs

```diff
@@ -191,6 +187,22 @@ impl Bytecode {
         self.write_op(OpcodeId::JUMPDEST);
         self.code.len()
     }
+
+    /// Store another code to memory
+    pub fn store_code_to_mem(&mut self, code: &Self) {
+        let len = code.codesize();
+        // pad to multiple of 32 bytes
+        let code: Vec<u8> = code
+            .code()
+            .iter()
+            .cloned()
+            .chain(0u8..((32 - len % 32) as u8))
```

Comment:
I keep the original code from tracer tests. But I keep wondering if we should pad only zeros but not `0...something`